backend/pkg/proto/service.go

```diff
@@ -412,15 +412,12 @@ func (s *Service) createProtoRegistry() error {
 // protoFileToDescriptorWithBinary parses a .proto file and compiles it to a descriptor using the protoc binary. Protoc must
 // be available as command or this will fail.
 // Imported dependencies (such as Protobuf timestamp) are included so that the descriptors are self-contained.
-//
-// ProtoPath is the path that contains all .proto files. This directory will be searched for imports.
-// Filename is the .proto file within the protoPath that shall be parsed.
 func (s *Service) protoFileToDescriptor(files map[string]filesystem.File) ([]*desc.FileDescriptor, error) {
 	filesStr := make(map[string]string, len(files))
 	filePaths := make([]string, 0, len(filesStr))
 	for _, file := range files {
 		filesStr[file.Path] = string(file.Payload)
-		filePaths = append(filePaths, file.Path)
+		filePaths = append(filePaths, string(file.Path[1:len(file.Path)]))
```

Comment:
```suggestion
		filePaths = append(filePaths, file.Path[1:])
```

That should be fine :)

Reply:
Nice, this looks much better, applied this